tl/controllers/middleware.rst

```diff
@@ -72,55 +72,55 @@ process, you can use the ``MiddlewareQueue`` object to attach middleware::
     {
         public function middleware($middlewareQueue)
         {
-            // Bind the error handler into the middleware queue.
+            // Itali ang tagahawak ng pagkakamali sa middleware na pila.
             $middlewareQueue->add(new ErrorHandlerMiddleware());
             return $middlewareQueue;
         }
     }
 
-In addition to adding to the end of the ``MiddlewareQueue`` you can do
-a variety of operations::
+At saka sa pagdaragdag sa katapusan sa ``MiddlewareQueue`` maaari kang makagawa
+ng iba't ibang mga operasyon::
 
         $layer = new \App\Middleware\CustomMiddleware;
 
-        // Added middleware will be last in line.
+        // Idinagdag na middleware ay magiging huli sa linya.
         $middlewareQueue->add($layer);
 
-        // Prepended middleware will be first in line.
+        // Inihanda na middleware ay magiging una sa linya.
```

Comment:
Inihanda is not the tagalog for prepended. Instead use `Ang idinagdag sa unahan`

Reply:
done change